packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -57,82 +32,51 @@ export const Dropdown = forwardRef(
             items,
             content,
             isDisabled,
-            addon,
             placement,
-            onToggle,
-            children,
-            icon = 'dotsThree',
+            iconName = 'dotsThree',
             'data-testid': dataTest,
         }: DropdownProps,
         ref,
     ) => {
-        const [isToggled, setIsToggledState] = useState(false);
-
         const popoverRef = useRef<PopoverRef>(null);
         const menuRef = useRef<HTMLUListElement>(null);
-        const toggleRef = useRef<HTMLDivElement>(null);
-
-        useEffect(() => {
-            if (!isToggled) {
-                toggleRef.current?.blur();
-            }
-
-            // focus the menu when it's toggled and there is content, not items
-            if (isToggled && content) {
-                menuRef.current?.focus();
-            }
-        }, [isToggled, content]);
-
-        const setToggled = (isToggled2: boolean) => {
-            if (onToggle) onToggle(isToggled2);
-            setIsToggledState(isToggled2);
-        };
 
         useImperativeHandle(ref, () => ({
             close: () => {
-                setToggled(false);
                 popoverRef.current?.close();
             },
+            open: () => {
+                popoverRef.current?.open();
+            },
         }));
 
-        useOnClickOutside([menuRef, toggleRef], () => {
```

Comment:
Lame question: Is this handled somehwere properly? IT works well, but I can't see where is it handled.